libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SFSDKWebViewStateManager.h

```diff
@@ -24,7 +24,18 @@
 #import <Foundation/Foundation.h>
 
 @class WKProcessPool;
-@interface SFSDKWKProcessPoolFactory : NSObject
+
+@interface SFSDKWebViewStateManager : NSObject
+
+/**
+  Reset some associated state with WKWebView and UIWebView e.g. Remove Cookies then add SID cookie
+ */
++ (void)resetSessionWithAccessToken:(NSString *) accessToken andProtocol:(BOOL) isSecure;
```

Comment:
Couple of minor asks:

- Can this be `resetSessionWithNewAccessToken`?  The current wording makes it sound like you're resetting the session associated with the access token.
- `andProtocol` would be better described as e.g. `isSecureProtocol`.  The former makes it sound like you're passing in the protocol.